src/components/utils/src/threads/thread_posix.cc

```diff
@@ -299,6 +306,9 @@ void Thread::StopUnsafe() {
 
 Thread::~Thread() {
   if (0 != handle_) {
+#ifdef THREAD_COUNT
```

Comment:
@OHerasym please do it after thread had been destroyed (at the end of destructor)